Description: force the ullong conversion anywhere.
Author: Sylvestre Ledru <sylvestre.ledru@scilab-enterprises.com>
Index: hdf5/src/H5Tconv.c
===================================================================
--- hdf5.orig/src/H5Tconv.c
+++ hdf5/src/H5Tconv.c
@@ -8017,7 +8017,6 @@ H5T__conv_ullong_double (hid_t src_id, h
  *
  *-------------------------------------------------------------------------
  */
-#if H5T_CONV_INTERNAL_ULLONG_LDOUBLE
 herr_t
 H5T__conv_ullong_ldouble (hid_t src_id, hid_t dst_id, H5T_cdata_t *cdata,
 		       size_t nelmts, size_t buf_stride,
@@ -8026,7 +8025,6 @@ H5T__conv_ullong_ldouble (hid_t src_id,
 {
     H5T_CONV_xF(ULLONG, LDOUBLE, unsigned long long, long double, -, -);
 }
-#endif /*H5T_CONV_INTERNAL_ULLONG_LDOUBLE*/
 
 

 /*-------------------------------------------------------------------------
